webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/healthcheck/HealthcheckService.scala

```diff
@@ -0,0 +1,58 @@
+package com.normation.rudder.services.healthcheck
+
+import java.lang.Runtime.getRuntime
+
+import com.normation.errors.IOResult
+import com.normation.rudder.domain.logger.{HealthcheckLoggerPure => logger}
+import zio.UIO
+import zio.ZIO
+
+sealed trait Status
+case object Ok extends Status
+case object Warning extends Status
+case object Critical extends Status
+
+trait Check {
+  def name: String
+  def run: IOResult[HealthcheckResult]
+}
+
+final case class HealthcheckResult(
```

Comment:
Y do think we need the value not just a message. You will need to represent it correctly in your dashboard... But maybe they are juste green checks / red cross for now ...

Reply:
Yes, just green / yellow / red + explanation message (that will likely contains the expected value and the actual value). But not need for different vizualization for now.